peekingduck/pipeline/nodes/model/hrnet.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright 2021 AI Singapore
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from typing import Dict, Any
+from peekingduck.pipeline.nodes.node import AbstractNode
+from .hrnetv1 import hrnet_model
+
+
+class Node(AbstractNode):
+    """HRNet node class that initialises and use hrnet model to infer poses
+    from detected bboxes
```

Comment:
Help me add this information 

Deep High-Resolution Representation Learning for Visual Recognition
https://arxiv.org/abs/1908.07919

Reply:
okay, added thank you.